components/net/filemanager_thread.rs

```diff
@@ -178,7 +178,7 @@ impl<UI: 'static + UIProvider> FileManager<UI> {
                     if let Ok(id) = Uuid::parse_str(&id.0) {
                         spawn_named("revoke blob url".to_owned(), move || {
                             // Since it is revocation, unset_url_validity is true
-                            let _ = sender.send(store.dec_ref(&id, &origin, true));
+                            let _ = sender.send(store.set_blob_url_validity(false, &id, &origin));
```

Comment:
The idea is that as long as `Blob` is not dropped or `close`d, it holds a readable reference to resource by `FileID`. Thus, revocation should not `dec_ref` but just simply flip the validity bit.